data/seeds/000_truncate.js

```diff
@@ -1,4 +1,9 @@
 exports.seed = async function (knex) {
-  await knex.raw('TRUNCATE TABLE profiles CASCADE');
```

Comment:
We'll have to use cascade to truncate any table that involve a foreign key.  Again that format is compatible with PostgreSql.